### DISCUSSION TITLE
❌ E-commerce improvement

### DISCUSSION THREAD

**Discussion by Alexisalvc** [elementor/elementor#23781] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi, I've noticed over the last few years that the e-commerce part of Elementor is really limited. 

What is available to us is no longer up to date to be honest. Given that this is a sector that is constantly evolving, what we are given does not allow us to be competitive. We're forced to use a lot of add-ons, which aren't great either.

Doing e-commerce since 2017 and using Elementor Pro for 4 years, I see that we are falling further and further behind.

### Describe the solution you'd like

1- The first thing would be to modify the "product image" block. We'd need to be able to choose between positioning the product gallery images at the bottom of the product image or on the left. 

This would give the customer a quicker overview without having to scroll too much. Better user experience.

2- Have the possibility to add a video at the end of the product gallery. This is something a lot of big sites do, and customers love it. Once you've seen a few photos, you can then view the product video. 

3- Native Bundles. It's something that's important, especially in the current inflationary climate where people want to save money.

Having a bundle that can be used for multiple products or multiple quantities. This will be a game changer if we can use it as a bundle for a number of products or to choose different quantities/containers of a product. Be able to see the product name, capacity or quantity, price and photo.

4- Disassociate add-to-cart button from quantity. It should be two separate buttons. This would make product page design much easier.

5- A native option should be available to display the product price on the add-to-cart button. This is something that works very well, and is even more important when the product has variations or swatches.

6- The slide cart is far too simple and not customizable enough. For example, we'd have to add a bar to indicate the progress of the basket to qualify for free delivery, add a carousel for upsells, choose the buttons we want inside, possibility to add trust badge/icons bellow checkout button etc.

If required, I can provide photos to illustrate what I'm saying. I still have a lot to say about improvements, but I didn't know if I could do it. 

Alexis.

**Comment by Alexisalvc** [comment]
I add other features:

7- Add a widget for estimated delivery. This lets customers know that if they order on Wednesday the 17th, they will receive their parcel on Friday the 19th, for example.

8- Have native SVG payment icons, with the official colors like Visa, Mastercard, AMEX, Apple Pay for example. This is very common on another CMS I use, and placed under the ATC, it's super useful for reassuring customers.

9- Add the mini-cart free-shipping bar. Example: 17€ to go before getting free delivery. This increases the average shopping basket.

10- Add native address search at checkout. If I type 5 Street of Ne for example, it will find the full name, country and zip code so the customer doesn't have to type it all. 

11- Add the option: read more and less in an accordion. On a product page, when you have an accordion with the full product description, rather than having 20 lines the customer can read the first 4 and if he wants to read the rest he clicks on read more. Premium Addons for Elementor have this feature but add a plugin for 1 widget, it's not worth it.

12- A sticky add to cart with product name, whether in stock or not, price, quantity and ATC. With the option of modifying fonts, colors and borders as desired. It would also have to be optimized for products with variations, which is very rarely the case with ATC, and even worse on mobiles.

13- The ability to sort products in the shop pages. For CBD oil, for example, if you have 4 products, 5% oil, 10% oil, 20% oil and 30% oil, the 5% oil will rank last because the 5 comes after the 3. If you create the order manually, it will take longer, but you'll be able to classify them as you wish.

14- Native Ajax filters for shop and catalogue page. A simple way to classify products based on certain characteristics.

15- Have a widget with a bar that scrolls text inside. For example, put: Made in Europe, free delivery on orders over €50, etc., and let the text scroll endlessly. We can choose the color of the bar and change the text as we like.

16- More freedom to customize the product grid. Like the possibility of having the add to cart appear only when you hover over the image, at the bottom of the image, so as not to overload the grid with 4 flashy add to cart... Have the option of adding reviews, a little description.

**Comment by alexanderpankart** [comment]
It would be nice if there was a way that the WooCommerce product variations could be at least "somewhat" designed. Currently, this is rather random or theme-dependent and runs under the "shopping cart" widget.

**Comment by nicholaszein** [comment]
Hello,

thank you for sharing your interest and feedback on improving the compatibility with WooCommerce.

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

As required and explained in our [Contribution Guidelines](https://github.com/elementor/elementor/blob/main/.github/CONTRIBUTING.md#how-to-request-a-feature), and in the Feature Request form you submitted, we cannot accept Feature Requests with more than one request per thread. 

For this reason I'm closing this discussion.

Grateful for your understanding,

Kind regards
